config.js

```diff
@@ -102,7 +102,7 @@ function sendConfig() {
             "steam_db_link": $("#steam_db_link")[0].checked,
             "copy_link_button": $("#copy_link_button")[0].checked,
             "dynamic_function": $("#dynamic_function")[0].checked,
-            "colorize_friends": $("#colorize_friends")[0].checked,
+            "colorize_friends_me": $("#colorize_friends_me")[0].checked,
```

Comment:
That should be `value` not `checked`, it's `select` object

Reply:
true